ci/pipeline.yml

```diff
@@ -0,0 +1,134 @@
+---
+jobs:
+- name: logsearch-s3archiving-check-b.cld.gov.au
+  serial: true
+  plan:
+  - do:
+    - get: 5m
+      trigger: true
+    - task: check-bucket
+      params:
+        BUCKET_NAME: {{b-cld-bucket-name}}
+        AWS_ACCESS_KEY_ID: {{aws-access-key-id}}
+        AWS_SECRET_ACCESS_KEY: {{aws-secret-access-key}}
+        AWS_DEFAULT_REGION: {{aws-default-region}}
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: govau/cga-cf-bosh-cli
+        run:
+          path: sh
+          args:
+          - -exc
+          - aws s3api list-objects --bucket $BUCKET_NAME --prefix $(date -u --date="-20 minutes" +%Y/%m/%d/%H/%M) | grep Size
```

Comment:
I like doing inline tasks like this for one-off tasks, but in this case since you're repeating the task it would DRY up the pipeline having separate task yml and sh files...

Reply:
It's also nice to have external tasks as you can run test them easily with `fly execute`

Reply:
yep, agreed.  i was just lazy with trying it out and wanted to avoid creating a repo and checking files in.  Now that it's mostly there i'll create the task file and git resource.

Reply:
moved to task file.